### ISSUE TITLE
MIDI Aux Send is displayed as Audio Aux Send

### ISSUE THREAD

**Issue by suedwestlicht** [rncbc/qtractor#477]
Recent Qtractor (commit 9001e8cfca70c5332566ac0e32569ceabd4d5be2 )

MIDI Aux Send should be displayed as (MIDI) but it's displayed as (Audio).

![a](https://github.com/user-attachments/assets/4482bc01-339d-43b2-9adf-64c51598fae9)

**Comment by rncbc** [comment]
fixed in 3369068

thanks

**Comment by suedwestlicht** [comment]
Yes, but now each Qtractor session is marked dirty/changed when loaded.

**Comment by rncbc** [comment]
> Yes, but now each Qtractor session is marked dirty/changed when loaded.

should be something else... unrelated to the last two commits, 9001e8c or 3369068 ... also, not reproducible here: all sessions are loaded clean.

**Comment by suedwestlicht** [comment]
I found the reason. It's the recent Yoshimi version from Git. Can you fund out which action of Yoshimi makes Qtractor think that the session is dirty? Then I'll make a bug report for Yoshimi.

**Comment by suedwestlicht** [comment]
I opened an issue for Yoshimi.

https://github.com/Yoshimi/yoshimi/issues/209

**Comment by suedwestlicht** [comment]
#477 is fixed, so I close this issue.

Thx, Rui.
